test/app-forms.spec.js

```diff
@@ -308,6 +312,22 @@ describe('forms that have caused bugs', () => {
     expect(result.errors).to.be.empty;
   });
 
+  it('#234 - fill datetime field', async () => {
+    const result = await harness.fillForm('bug_234', ['2023-01-01 1:22']);
+    expect(result.errors).to.be.empty;
+  });
+
+  it('#234 - fill datetime expected input format', () => {
+    return expect(harness.fillForm('bug_234', ['2023-01-01']))
+      .to.be.rejectedWith('expect input in format');
```

Comment:
This error message is just for the test and not what is actually thrown, yes?

Reply:
[This](https://github.com/medic/cht-conf-test-harness/pull/238/files#diff-2595fb199699a5c7b868eb0b0dbfe4d80a599fd7b13c4cf8740c0139ed1e1c7cR204) is the actual error throw. The assertion here confirms the substring is present.